includes/library/zencart/admin/Controllers/src/AbstractController.php

```diff
@@ -56,20 +57,16 @@ public function __construct($controllerCommand, $request, $db)
     /**
      *
      */
-    public function prepareCommonTemplateVariables()
+    protected function prepareCommonTplVars()
```

Comment:
line 52 uses `$this->prepareCommontplVars()` ... and since this method signature uses a capital `T` I suspect line 52 should match?